src/lib/apiTypesV2/Entity.cpp

```diff
@@ -128,7 +136,30 @@ void Entity::fill(const std::string& _id, const std::string& _type, const std::s
   attributeVector.fill(aVec);
 }
 
+void Entity::fill(QueryContextResponse* qcrsP)
+{
 
+  if (qcrsP->errorCode.code != SccOk)
+  {
+    //
+    // If no entity has been found, we respond with a 404 NOT FOUND
+    // If any other error - use the error for the response
+    //
+    errorCode.fill(qcrsP->errorCode);
```

Comment:
This comment is a little bit strange, makes me think that code is missing.
Perhaps the comment should be outside 'if (qcrsP->errorCode.code != SccOk)' ?